### PR TITLE
Add Copy button to white label UX

### PR DESCRIPTION
**Pull request by compulim**
### Added

-  Added "Copy" button to bot messages in Fluent UI if it contains keyword `AllowCopy`, in PR [#5259](https://github.com/microsoft/BotFramework-WebChat/pull/5259) and [#5262](https://github.com/microsoft/BotFramework-WebChat/pull/5262), by [@compulim](https://github.com/compulim)

## Description

Ported the "Copy" button to both white-label and Fluent/Copilot theme.

## Design

The "Copy" button would show up if:

- An activity with type `message`, and;
- the activity with a non-empty `text` field, and;
- the activity has a `Message` thing in `entities` field with `@id` of empty string, and;
- the `Message` thing has `keywords` containing `"AllowCopy"`

If the `textFormat` is unset or `"markdown"`, both plain text and HTML text version of the message will be copied to the clipboard.

For example,

```json
{
  "entities": [
    {
      "@context": "https://schema.org",
      "@id": "",
      "@type": "Message",
      "keywords": ["AllowCopy"],
      "type": "https://schema.org/Message"
    }
  ],
  "text": "Hello, World!",
  "type": "message"
}
```

## Specific Changes

- Port the Fluent version of "Copy" button to white-label
- Reimplement Fluent/Copilot theme

-  [x] I have added tests and executed them locally
-  [x] I have updated `CHANGELOG.md`
-  [x] ~I have updated documentation~

## Review Checklist

> This section is for contributors to review your work.

-  [x] ~Accessibility reviewed (tab order, content readability, alt text, color contrast)~
-  [x] ~Browser and platform compatibilities reviewed~
-  [x] CSS styles reviewed (minimal rules, no `z-index`)
-  [x] ~Documents reviewed (docs, samples, live demo)~
-  [x] ~Internationalization reviewed (strings, unit formatting)~
-  [x] ~`package.json` and `package-lock.json` reviewed~
-  [x] ~Security reviewed (no data URIs, check for nonce leak)~
-  [x] Tests reviewed (coverage, legitimacy)

### CODE REVIEW

File: packages/component/src/index.ts [hunk 1]
```diff
@@ -103,23 +105,27 @@ const Components = {
   UploadButton,
 
   // Components for localization
-  LocalizedString
+  LocalizedString,
+
+  // Components for theming
+  MonochromeImageMasker
```
Comment (OEvgeny) [inline packages/component/src/index.ts:111]: Should we include some docs?

**Comment by OEvgeny** [comment]
Solid work. It looks like it should be done this way :rocket:

**Review by OEvgeny — APPROVED** [review]

File: packages/component/src/Styles/StyleSet/ActivityButton.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+export default function createActivityButtonStyle() {
+  return {
+    '&.webchat__activity-button': {
+      alignItems: 'center',
+      appearance: 'none',
+      background: '#fff',
+      borderRadius: 4,
```
Comment (OEvgeny) [inline packages/component/src/Styles/StyleSet/ActivityButton.ts:7]:
```suggestion
      borderRadius: '4px',
```
To make units consistent in the file

File: packages/component/src/Styles/StyleSet/ActivityButton.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      '&:active': {
+        background: '#e0e0e0',
+        border: '1px solid #b3b3b3',
+        color: '#242424'
+      },
+
+      '&:focus-visible': {
+        background: '#fff',
+        outline: '2px solid #000',
+        outlineOffset: -2
```
Comment (OEvgeny) [inline packages/component/src/Styles/StyleSet/ActivityButton.ts:30]:
```suggestion
        outlineOffset: '-2px'
```

File: packages/component/src/Styles/StyleSet/ActivityButton.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+      '&:disabled': {
+        background: '#f0f0f0',
+        border: '1px solid #e0e0e0',
+        color: '#bdbdbd'
+      },
+
+      '& .webchat__activity-button__icon': {
+        height: 20,
+        width: 20
```
Comment (OEvgeny) [inline packages/component/src/Styles/StyleSet/ActivityButton.ts:41]:
```suggestion
        height: '20px',
        width: '20px'
```

I just think it is easier to reason about. E.g.:
```
  fontSize: 16,
  lineHeihgt: 1
```
vs
```
  fontSize: '16px',
  lineHeihgt: 1
```

File: packages/component/src/Utils/MonochromeImageMasker.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+import React, { memo, useMemo, type CSSProperties } from 'react';
+import { useStyleSet } from '../hooks';
+
+type Props = Readonly<{ className?: string | undefined; src: string }>;
+
+// TODO: This need to be deduplicated with Fluent Theme.
```
Comment (OEvgeny) [inline packages/component/src/Utils/MonochromeImageMasker.tsx:7]: Looks like it is done?
```suggestion
```

File: packages/component/src/index.ts (hunk 1 above)
Comment (compulim) [reply packages/component/src/index.ts:111]: Oh, maybe I should tag it as `internal`.

Comment (compulim) [reply packages/component/src/index.ts:111]: Can't put it in `internal` yet as Jest run OOM while loading `botframework-webchat-api`.

**Review by OEvgeny — APPROVED** [review]

**Pull request merged**